### PR TITLE
ADEN-11534 Adding ad slot iframe resizing for new UAP companions

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Review by jakubbanas — APPROVED** [review]

File: platforms/shared/slots/slots-context.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {
 	AdSlot,
 	communicationService,
-	context,
+	context, Dictionary,
```
Comment (nandy-andy) [inline platforms/shared/slots/slots-context.ts:4]: Shouldn't we put it in the next line?

File: platforms/ucp-desktop/ad-context.ts [hunk 2]
```diff
@@ -116,7 +116,7 @@ export const basicContext = {
 	src: ['gpt'],
 	state: {
 		adStack: [],
-		isMobile: true,
+		isMobile: false,
```
Comment (nandy-andy) [inline platforms/ucp-desktop/ad-context.ts:119]: 💎

File: platforms/shared/slots/slots-context.ts (hunk 1 above)
Comment (dmnsobczak) [reply platforms/shared/slots/slots-context.ts:4]: What, why Prettier didn't fixed that :o FIxed!

File: platforms/ucp-desktop/ad-context.ts (hunk 2 above)
Comment (dmnsobczak) [reply platforms/ucp-desktop/ad-context.ts:119]: <3

File: src/ad-products/templates/uap/constants.ts [hunk 3]
```diff
@@ -21,9 +21,11 @@ export const UAP_ADDITIONAL_SIZES: Dictionary<Dictionary<[number, number]>> = {
 	desktop: {
 		bfaSize: [3, 3],
 		companionSize: [5, 5],
+		companionOriginalSize: [300, 600],
```
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/ad-products/templates/uap/constants.ts:24]: Here's the size map:
https://docs.google.com/spreadsheets/d/13RLrkB7Tp2AzXPXNWsPUdVRhV459uTpXggct6cvH3pM/edit#gid=0
300x250 are still targeted via 300x250 size. 5x5 is only for the big ones.

[the remark this answers is not in the document]
Comment (nandy-andy) [reply src/ad-products/templates/uap/constants.ts:24]: Consider putting it to an [ADR](https://medium.com/fandom-engineering/keep-programmers-decisions-documentation-up-to-date-and-simple-with-architecture-decision-records-8ef5f1761ba)... maybe in `src/ad-products/templates/uap/ADR/` ;)

**Pull request merged**